src/ReflectionInterface.php

```diff
@@ -0,0 +1,15 @@
+<?php
+/**
+ * Parser Reflection API
+ *
+ * @copyright Copyright 2015, Lisachenko Alexander <lisachenko.it@gmail.com>
+ *
+ * This source file is subject to the license that is bundled
+ * with this source code in the file LICENSE.
+ */
+namespace Go\ParserReflection;
+
+interface ReflectionInterface
```

Comment:
@lisachenko, I'm kind of knee-deep in alligators here, trying to finish and untangle this into two workable PRs here, but I wanted to give you an update relevant to the `Go\ParserReflection\ReflectionInterface` interface I added. I added it believing that `\Reflection` was the parent class of all Reflection classes. I have since discovered (at least in PHP 7) this isn't the case. The `\Reflector` interface, however **is** implemented by all Reflection classes (except, ironically, the purely-static `\Reflection` class). So the new interface `Go\ParserReflection\ReflectionInterface` should clearly be renamed either `Go\ParserReflection\ReflectorInterface` or simply `Go\ParserReflection\Reflector` (as `\Reflector` is already an interface). I welcome your input.

I am continuing work on this PR here: [loren-osborn:losborn_testDirReorgAndPassThrough](https://github.com/loren-osborn/parser-reflection/tree/losborn_testDirReorgAndPassThrough) which will be split into two distinct PRs.